### PR TITLE
Server hosting but cooler

### PR DESCRIPTION
**Pull request by NoCatt**
Removes the convars from random sections and gives settings their own section.
also adds a hosting on windows section to better distinguish between the 2 form of deficated server

### CODE REVIEW

File: docs/hosting-a-server-with-northstar/dedicated-server/hosting-on-windows.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+## Minimum requirements
+
+The current minimum requirements are as follows:
+
+**Requirements per server:**
+
+- 8GB free disk space (5GB of the game and 3 for Origin/EA app)
+- 3+ cores (2 might work, though)
+- 3GB+ total memory (RAM or swap will do)
+- Windows 8.1 or better (and built-in WARP [use `-softwared3d11`] or a graphics card)
```
Comment (itscynxx) [inline docs/hosting-a-server-with-northstar/dedicated-server/hosting-on-windows.md]:
```suggestion
- Windows 8.1 or better (and built-in WARP using the [`-software3d11` launch arg](./#dedicated-server-caveats), or a physical graphics card
```

File: docs/hosting-a-server-with-northstar/dedicated-server/hosting-on-windows.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+**Note:** It is recommended to surpass the listed requirements. Currently the number of available servers covers the daily playerbase more than enough. If you're planning to host public servers for the community it is therefore recommended to either fill a niche (like a gamemode that is so popular that all existing servers are full) or provide a better service than existing hosts (less lag, more stable, etc.).
+
+
+## Setup
+
+Dedicated servers allow you to host a Northstar server without having to use a full client, making them more lightweight and easier to host for longer periods of time. Dedicated servers are still in development for Northstar, so while they do work, expect some bugs and general jank.\
+To start a dedicated server on Northstar, launch NorthstarLauncher.exe with the argument `-dedicated`, this can be done manually, but releases also provide the batch file `r2ds.bat`, which will also do this.\
+When using a dedicated server, arguments are read from `ns_startup_args_dedi.txt`, rather than `ns_startup_args.txt`.
+
```
Comment (itscynxx) [inline docs/hosting-a-server-with-northstar/dedicated-server/hosting-on-windows.md:33]:
```suggestion

You can also follow a video guide for this process from [this YouTube link](https://www.youtube.com/watch?v=EZ3w2Nl9SZo)

```
if doing this, also delete the entire `guides` folder from `docs/hosting-a-server-with-northstar/`, and remove line 46 from [`SUMMARY.md`](https://github.com/NoCatt/NorthstarWiki/blob/re-serverhosting/docs/SUMMARY.md?plain=1#L46)

**Review by itscynxx — CHANGES_REQUESTED** [review]
A couple smaller things I see from a first read

File: docs/hosting-a-server-with-northstar/dedicated-server/hosting-on-windows.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+Regarding RAM usage, dedicated servers also use significant amounts of RAM at the moment, often requiring 1.5-2GB, though I expect this to lower as development continues.
+
+### Video guide
+
+{% hint style="info" %}
+Note that these guides might not fully reflect best practices and can be outdated. However they should be helpful in setting up a server for the first time.\
+If you're planning to get serious about hosting your own server, please check out the ["Hosting a Dedicated Server" section](../dedicated-server/).
+{% endhint %}
+
+{% embed url="https://www.youtube.com/watch?v=EZ3w2Nl9SZo" %}
```
Comment (GeckoEidechse) [inline docs/hosting-a-server-with-northstar/dedicated-server/hosting-on-windows.md:60]: Not sure if we should even link to it any more as it's severely outdated. Maybe with a big asterisk that TCP forwarding is no longer a thing?

Comment (NoCatt) [reply docs/hosting-a-server-with-northstar/dedicated-server/hosting-on-windows.md:60]: will do

Comment (NoCatt) [reply docs/hosting-a-server-with-northstar/dedicated-server/hosting-on-windows.md:60]: let me know if you think this is good

**Comment by GeckoEidechse** [comment]
Merge conflicts atm

**Review by GeckoEidechse — COMMENTED** [review]
This is getting kinda too big to review (especially in regards to making sure that all links have been updated).

I think it makes most sense to make a few smaller PRs with the individual logical changes (adding Windows hosting section, removing RCON page, etc...)

Keep this PR open in the meantime still so that we have a target on what all we need to change ^^

**Comment by GeckoEidechse** [comment]
With #218 merged this PR here obviously merge conflicts now. @NoCatt LMK when you have looked into it and I'll give the PR a look ASAP ^^

**Comment by NoCatt** [comment]
split it into differen PRs

**Pull request closed without merging**